skl2onnx/operator_converters/RandomForest.py

```diff
@@ -13,6 +13,17 @@
 from ..common.tree_ensemble import get_default_tree_regressor_attribute_pairs
 
 
+def _num_estimators(op):
```

Comment:
Why do you need a separate function for this?

Reply:
My thinking goes like this... I like to comment key decisions like this for the next person. Do I put the same comment in two places? Or "factor" the comment out to a separate function? I tend to take the latter for maintenance reasons in projects I work on though I recognize that is a stylistic decision. I'm ok changing this if that is what you strongly prefer.

Reply:
I am not sure if you need the comments here. In case you feel it's necessary, then yes this seems all right.